lib/Sema/DerivedConformances.cpp

```diff
@@ -203,6 +210,14 @@ ValueDecl *DerivedConformance::getDerivableRequirement(TypeChecker &tc,
 
     // Retrieve the requirement.
     auto results = proto->lookupDirect(name);
+    // SWIFT_ENABLE_TENSORFLOW
+    // Filter requirements, if `filter` function is specified.
```

Comment:
Note: adding a `filter` function here is necessary because, for the first time, derived conformances must handle two protocol requirements with the same name: `pow(_ x: Self, _ y: Self) -> Self` and `pow(_ x: Self, _ n: Int) -> Self`.